RecoDataProducts/inc/STMMWDDigi.hh

```diff
@@ -0,0 +1,32 @@
+#ifndef RecoDataProducts_STMMWDDigi_hh
+#define RecoDataProducts_STMMWDDigi_hh
+//
+// Data product that represents an uncalibrated hit in the STM
+//
+
+// C++ includes
+#include <iostream>
+#include <vector>
+#include <array>
+#include <Rtypes.h>
+
+namespace mu2e {
+
+  class STMMWDDigi {
+  public:
+    STMMWDDigi() : _time(0), _energy(0){};
+
+    STMMWDDigi(uint32_t time, int16_t energy) : _time(time), _energy(energy) {};
+
+
+    uint32_t time() const { return _time; }
+    int16_t energy() const { return _energy; }
+
+  private:
+    uint32_t _time; // uncalibrated time of hit [ct]
+    int16_t _energy; // uncalibrated energy [ADC units]
+  };
+
```

Comment:
Consider adding sort_by_time and sort_by_energy as free functions.  ( Or spell them sortByTime and sortByEnergy); if they will ever be needed this is where they belong.   For a class like this I recommend not defining operator<  since it is ambiguous.

Reply:
Thanks for the suggestions. I have added these functions in commit [10753dc](https://github.com/Mu2e/Offline/pull/938/commits/10753dcb03444afd240e5c53f9f1f2616623c371) and did a quick test in the MWD module itself

Reply:
Thanks Andy.  I meant to ask you to name the functions lessByTime and lessbyEnergy.  Please make that change.

Reply:
Thanks, Rob. Done in commit [c54d7f0](https://github.com/Mu2e/Offline/pull/938/commits/c54d7f0a9c54a38ece58958be9cfaaa52d9c512e)